asciidoc/volume2/service-mapping/tf2-ch-a-mdpws-service-mapping.adoc

```diff
@@ -54,3 +52,11 @@ NOTE: `GetContextStatesByIdentification` and `GetContextStatesByFilter` are insu
 NOTE: A <<vol1_spec_sdpi_p_actor_somds_consumer>> may retrieve context states by using `GetContextStates` and perform filtering by itself.
 ====
 ****
+
+.R0509
+[sdpi_requirement#r0509,sdpi_req_level=shall]
+****
+If a <<vol1_clause_sdpi_p_somds_provider>> provides localized text references in its <<acronym_mdib>>, the <<vol1_clause_sdpi_p_somds_provider>> shall provide the LOCALIZATION SERVICE via port type ++{++{uri_sdc_port_type}++}++LocalizationService.

```

Comment:
In the artifact, this displays in a strange way ("1:10.1.1.2 SOMDS Provider" instead of "SOMDS Provider")... not sure why the same does not happen elsewhere (e.g., for the SOMDS Consumer in the preceding requirement).

Reply:
Oh yes, you're right, it is because I am referencing the clause rather than the actor... it is somewhat confusing, we have three different links to actors... IHE mysteries